widget/ui/package.json

```diff
@@ -26,7 +26,6 @@
   "peerDependencies": {
```

Comment:
You can use `PendingSwapNetworkStatus` from `rango-types` in `widget/ui/src/types/swaps.ts`

Reply:
fixed